dvc/external_repo.py

```diff
@@ -4,11 +4,11 @@
 import tempfile
 from contextlib import contextmanager
 from distutils.dir_util import copy_tree
-
 from funcy import retry
 
-from dvc.config import NoRemoteError
```

Comment:
Same here, no need to move it really. It breaks the git blame.

Reply:
@efiop codeclimate asked me to do it :-X

Reply:
@maykulkarni It is purely advisory, no need to make it fully pass.